### PR TITLE
force all queries to be lowercase bug 681747

### PR DESCRIPTION
**Pull request by pennyfx**

### CODE REVIEW

File: apps/search/views.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def search(request):
     form = SearchForm(request.GET)
     form.is_valid()
     query = form.cleaned_data
-    q = query.get('q')
+    q = query.get('q').lower()
```
Comment (seanmonstar) [inline apps/search/views.py:18]: Does query.get() sometimes return None, or always an empty string?

Comment (seanmonstar) [inline apps/search/views.py:18]: Sorry, it's from `form.cleaned_data`, so it's just a normal `dict`, meaing it can return None.

Comment (pennyfx) [inline apps/search/views.py:18]: Here's the output when a query is not defined.  It looks like it converts the CharField into an empty string.  It appears to be safe, but I can easily add a None check before doing .lower() 

`{'used': 0, 'author': None, 'copies': 0, 'q': u'', 'activity': None, 'type': u'', 'page': None}`

Comment (seanmonstar) [inline apps/search/views.py:18]: Perhaps the Form forces an empty string since its a text field. If 
that's the case, then I guess it's safe enough.

r+

**Pull request merged**